char_creation.php

```diff
@@ -7,52 +7,88 @@
 
 <html>
     <head>
-      <a href='project_index.php'>Go back</a>
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+	    <link href="normalize.css" type="text/css" rel="stylesheet" />
+	    <link href='https://fonts.googleapis.com/css?family=Montserrat' rel='stylesheet'>
+	    <link href="style.css" type="text/css" rel="stylesheet" />
+		    <title>Create Character</title>
     </head>
     <body>
-        <form action='verify.php' method='POST'>
-            <div>
-               <?= $error ?>
-            </div> 
-            <div>
-                <label for='charname'>Character Name:</label>
-                <input type='text' name='charname' />
+        <nav>
+            <div class='nav-links'>
+                <a href="project_index.php">Home</a>
             </div>
-            <div>
-                <label for='class'>Class:</label>
-                <input type='text' name='class' />
+            <div class='nav-contact'>
+                <a href="https://cr-walton.github.io/">Portfolio</a>
+                <a href="crwaltoncv.docx" id="cv">Download CV</a>
             </div>
-            <div>
-                <label for='level'>Level:</label>
-                <input type='number' min= '0' max= '20' name='level' />
+        </nav>
+        <h1>Create Your Character</h1>
+        <section class='form_container'>
+            <form class='edit_create_form' action='verify.php' method='POST'>
+                <div>
+                   <?= $error ?>
+                </div> 
+                <div>
+                    <label for='charname'>Character Name:</label>
```

Comment:
read the "Tips and Notes" section on this page:
https://www.w3schools.com/tags/tag_label.asp
all your labels need to change